From fb6a54c36a86dd039a394bb2b4fc93ad45bb8f95 Mon Sep 17 00:00:00 2001
From: vinit <vinitsiriya@gmail.com>
Date: Sun, 26 May 2024 20:41:40 +0530
Subject: [PATCH] bug fix : gpt automation worked

---
 gpt_automation/main.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gpt_automation/main.py b/gpt_automation/main.py
index a5e9116..13bcb57 100644
--- a/gpt_automation/main.py
+++ b/gpt_automation/main.py
@@ -56,7 +56,7 @@ def prompt_only_dir_structure(project_path='.', profile_name=None):
     with open(white_list_file, 'r') as f:
         white_list = f.read()
 
-    project_info = ProjectInfo(project_path, black_list, white_list)
+    project_info = ProjectInfo(project_path, black_list, white_list,profile_name)
 
     directory_structure_prompt = project_info.create_directory_structure_prompt()
     print(directory_structure_prompt)
-- 
2.45.1.windows.1

